draft-ietf-quic-transport.md

```diff
@@ -1351,6 +1355,14 @@ connection when they move networks.  This includes state that can be hard to
 recover such as outstanding requests, which might otherwise be lost with no easy
 way to retry them.
 
+An endpoint that receives packets that contain a source IP address and port that
```

Comment:
Ok, so this is explicitly to handle unintentional port/IP migration and intentional port/IP migration without changing paths(ie: privacy preservation)?  I wouldn't call that connection migration, which I think of as explicitly moving from one path to the other, but not using both paths at once.  I'll re-read this with that in mind.